gnomad_hail/resources/import_resources.py

```diff
@@ -38,45 +40,32 @@ def get_module_importable_resources(module, prefix: Optional[str] = None) -> Dic
     return resources
 
 
-def add_arguments_from_resources_dict(container: argparse._ActionsContainer, resources_dict: Dict[str, Tuple[str, BaseResource]]) -> None:
-    """
-    Adds resources as arguments to an argparse._ActionsContainer (typically a parser or argument group) from a resources dict as generated by get_module_importable_resources.
-    Each resource is added as argument `--resource_key` where resource_key is its corresponding key in `resources_dict`, using `action="store_true"`.
-
-    :param container: Input argparse container
-    :param resources_dict: Resources to add
-    :return: Nothing
-    """
-    for arg_name, (resource_name, resource) in resources_dict.items():
-        container.add_argument(
-            f"--{arg_name}",
-            help="Imports {}{} to {}".format(
-                resource_name,
-                " from " + resource.import_args.get('path', '???') if resource.import_args else "???",
-                resource.path
-            ),
-            action="store_true"
-        )
-        
-
 grch37_resources = get_module_importable_resources(grch37, 'grch37')
 grch38_resources = get_module_importable_resources(grch38, 'grch38')
+all_resources = {**grch37_resources, **grch38_resources}
 
 
 def main(args):
-    resources = grch37_resources.copy()
-    resources.update(grch38_resources)
-    for arg in [arg for arg, arg_value in vars(args).items() if arg_value and arg != 'overwrite']:
-            resource_name, resource = resources[arg]
-            print(f"Importing {resource_name}...")
-            resource.import_resource(args.overwrite)
+    for resource_arg in args.resources:
+        resource_name, resource = all_resources[resource_arg]
+        print(f"Importing {resource_name}...")
+        resource.import_resource(args.overwrite)
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser()
-    grch37_grp = parser.add_argument_group('grch37 resources')
-    add_arguments_from_resources_dict(grch37_grp, grch37_resources)
-    grch38_grp = parser.add_argument_group('grch38 resources')
-    add_arguments_from_resources_dict(grch38_grp, grch38_resources)
+    wrapper = textwrap.TextWrapper(width=100, initial_indent=" " * 2, subsequent_indent=" " * 4)
```

Comment:
The usage output here is significantly wider than argparse's default. Using `100` here allows most resource paths and import paths to fit on one line instead of being wrapped.